sql/core/src/test/scala/org/apache/spark/sql/SparkSessionExtensionSuite.scala

```diff
@@ -26,6 +26,7 @@ import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan
 import org.apache.spark.sql.catalyst.rules.Rule
 import org.apache.spark.sql.execution._
 import org.apache.spark.sql.execution.vectorized.OnHeapColumnVector
+import org.apache.spark.sql.internal.StaticSQLConf
```

Comment:
Can we change like the following if we need only one configuration import?
```
- import org.apache.spark.sql.internal.StaticSQLConf
+ import org.apache.spark.sql.internal.StaticSQLConf.SPARK_SESSION_EXTENSIONS
```

Then, the other will be shorter.
```
- .config(StaticSQLConf.SPARK_SESSION_EXTENSIONS.key, classOf[MyExtensions].getCanonicalName)
+ .config(SPARK_SESSION_EXTENSIONS.key, classOf[MyExtensions].getCanonicalName)
```